src/bin/asr_train.py

```diff
@@ -63,10 +63,16 @@ def main():
     parser.add_argument('--subsample', default=1, type=str,
                         help='Subsample input frames x_y_z means subsample every x frame at 1st layer, '
                              'every y frame at 2nd layer etc.')
-    # loss
-    parser.add_argument('--ctc_type', default='warpctc', type=str,
-                        choices=['chainer', 'warpctc'],
-                        help='Type of CTC implementation to calculate loss.')
+    # ctc
+    parser.add_argument('--ctype', default='warpctc', type=str,
+                        choices=['warpctc', 'chainer', 'blstm'],
+                        help='Type of CTC architecture.')
```

Comment:
The name `blstm` is a little bit confusing.
How about adding non-shared layer when `clayers > 0` while keeping `ctype=warpctc`?